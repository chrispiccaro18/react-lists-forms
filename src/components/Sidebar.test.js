import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('Sidebar component', () => {
  it('renders a Sidebar', () => {
    const wrapper = shallow(
      <Sidebar>
        <p>Testing</p>
      </Sidebar>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
