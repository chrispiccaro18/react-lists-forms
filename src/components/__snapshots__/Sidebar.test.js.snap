// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component renders a Sidebar 1`] = `
<nav>
  <p>
    Testing
  </p>
</nav>
`;
